### PR TITLE
Don't recommend SVG, remove gmail message

### PR DESCRIPTION
**Pull request by rijkvanzanten**
Having edge case warnings like this in the docs are very hard to maintain, and will add quite a bit of cruft. We should instead have recommended approaches that prevent those edge cases from becoming real problems 👍🏻

### CODE REVIEW

**Comment by erondpowell** [comment]
> Having edge case warnings like this in the docs are very hard to maintain...

Yeah you're right, and we have already discussed handling these edge cases with the "in-docs GH discussions" plan.
Closing now.

**Comment by rijkvanzanten** [comment]
@erondpowell That's not the same. This PR is to cleanup the messaging in the current docs, which I'd very much like to merge through. 

I don't want this:

CleanShot 2022-08-18 at 13 35 20@2x

As using both SVGs _and_ Gmail web client simultaneously is not a given for every user, and is therefore an edge case that I'd rather not explicitly warn for. Instead, we should update the messaging above to _not_ recommend using SVGs:

CleanShot 2022-08-18 at 13 36 03@2x

---

TL;DR: Instead of "We recommend SVG/PNG + Don't use SVG cause it doesn't work in Gmail" it becomes "We recommend PNG"

**Comment by erondpowell** [comment]
> @erondpowell That's not the same. This PR is to cleanup the messaging in the current docs, which I'd very much like to merge through.
> 
> I don't want this:
> 
> CleanShot 2022-08-18 at 13 35 20@2x
> 
> As using both SVGs _and_ Gmail web client simultaneously is not a given for every user, and is therefore an edge case that I'd rather not explicitly warn for. Instead, we should update the messaging above to _not_ recommend using SVGs:
> 
> CleanShot 2022-08-18 at 13 36 03@2x
> 
> TL;DR: Instead of "We recommend SVG/PNG + Don't use SVG cause it doesn't work in Gmail" it becomes "We recommend PNG"

okay, sounds good! I see your commit and I'll merge this then.
